packages/EasyCodingStandard/README.md

```diff
@@ -240,6 +240,30 @@ parameters:
     line_ending: "\n"
 ```
 
+## Integration
+
+### PHPStorm
+
+EasyCodingStandard can be used as an External Tool
+
+![PHPStorm Configuration](docs/phpstorm-config.png)
+
+Go to `Preferences` > `Tools` > `External Tools` and click `+` to add a new tool.
+
+- Name: `ecs` (Can be any value)
+- Description: `easyCodingStandard` (Can be any value)
+- Program: `$ProjectFileDir$/vendor/bin/ecs` (Locally or Globally installed ecs executable)
+- Parameters: `check $FilePathRelativeToProjectRoot$` (append `--fix` to auto-fix)
+- Working directory: `$ProjectFileDir$`
+
+> On Windows path separators must be `\` in Program field.
+
+Press `Cmd/Ctrl` + `Shift` + `A` (Find Action), search for `ecs`, and then hit Enter.
+
+It will run `ecs` for the current file. To run `ecs` on a directory, right click on a folder in the project browser go to external tools and select `ecs`.
+
+You can add a keyboard shortcut in [Preferences > Keymap](https://www.jetbrains.com/help/webstorm/configuring-keyboard-and-mouse-shortcuts.html) to run `ecs` configuration.
```

Comment:
Which of those 2 options are more common?

Reply:
It depends on personal workflow.

I prefer the keyboard. Someone might like to use a mouse or the action finder.